druid/datadog_checks/druid/data/conf.yaml.example

```diff
@@ -46,9 +46,9 @@ init_config:
 instances:
 
     ## @param url - string - required
-    ## url used to connect to the Druid instance.
+    ## coordinator url used to connect to the Druid instance.
     #
-  - url: <DRUID_INSTANCE_URL>
+  - url: <DRUID_COORDINATOR_URL>
```

Comment:
The `conf.yaml.example` is not meant to be modified by hand but generated from the [spec.yaml](https://github.com/DataDog/integrations-core/blob/master/druid/assets/configuration/spec.yaml) file with the `ddev validate config druid -s`. Could you also update this one?